src/vue/modules/create-asset-form/components/advanced-step-form.vue

```diff
@@ -121,18 +128,19 @@ const EVENTS = {
 
 export default {
   name: 'advanced-step-form',
+  components: { VueMarkdown },
   mixins: [FormMixin],
   props: {
     request: { type: CreateAssetRequest, default: null },
     isDisabled: { type: Boolean, default: false },
-    accountId: { type: String, required: true },
+    publicKey: { type: String, required: true },
```

Comment:
does not describe actual use of the prop, confusing with current terminology (account id is preferable).

Reply:
@max-nes why? the string we pass to the prop is exactly the public key and not the account ID

Reply:
@quinn71 
1. Actually every account id in the client is public key and being assigned from a newly created signer, is not it?
2. You get it from `this.wallet.keypair.accountId()`
3. This is not the first occurrence of wrongly terminology used wrongly: seed is also should be called "secret key" or "private key". If we fucked up one day with wrongly named method `accountId()` of keypair that actually returns a public key, we should either follow this messy way to the end of time OR we should repair all of the occurrences to bring consistency, so we won’t call the same thing with two different terms like we did for funds and tokens lately.
4. Consistency

Reply:
@ardier16 we agreed on "signerAccountId", but it still does not describe role the prop takes

Reply:
Ok, renamed